libs/application/templates/marriage-conditions/src/lib/getApplicationFeatureFlags.ts

```
@@ -1,32 +1,32 @@
+      featureFlags.map(async (key: MarriageConditionsFeatureFlags) => {
         return { key, value: !!(await client.getValue(key, false)) }
       }),
     )
   ).reduce(
     (
       acc,
-      { key, value }: { key: MarriageCondtionsFeatureFlags; value: boolean },
+      { key, value }: { key: MarriageConditionsFeatureFlags; value: boolean },
     ) => {
       return {
         ...acc,
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)